example-form/components/FormRegisterPetDetails/FormRegisterPetDetailsStep1.tsx

```diff
@@ -129,19 +129,13 @@ export const FormRegisterPetDetailsStep1 = () => {
 					<Controller
 						control={control}
 						name="dob"
-						render={({
-							field: { onChange, onBlur, value, name, ref },
-							fieldState: { invalid, error },
-						}) => (
+						render={({ field: { ref, ...field } }) => (
```

Comment:
This fixes a bug where the invalids state does not get shown to the user. Will do some further investigation after this PR is merged, but for now this fixes the issue.